cdc/processor/agent.go

```diff
@@ -166,6 +192,19 @@ func (a *agentImpl) FinishTableOperation(
 	tableID model.TableID,
 	epoch model.ProcessorEpoch,
 ) (done bool, err error) {
+	topic := model.SyncTopic(a.changeFeed)
+	if !a.Barrier(ctx) {
+		if _, exists := a.barrierSeqs[topic]; exists {
+			log.L().Info("Delay sending FinishTableOperation "+
+				"due to pending sync",
```

Comment:
```suggestion
			log.L().Info("Delay sending FinishTableOperation due to pending sync",
```

It's okay, as long as it does not exceed 100.